src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/pipeline_management.yml

```diff
@@ -0,0 +1,879 @@
+              commands:
+                - cdk --version
+                - mkdir cdk_inputs
+                - cp definition.json cdk_inputs/definition.json
+                - cdk synth --app adf-build/cdk/generate_pipeline_stacks.py -vv
+                - python adf-build/cdk/execute_pipeline_stacks.py
+      ServiceRole: !GetAtt PipelineManagementCodeBuildProjectRole.Arn
+  PipelineManagementCodeBuildProjectRole:
+    Type: AWS::IAM::Role
+    Properties:
+      Path: "/adf-automation/"
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - codebuild.amazonaws.com
+            Action:
+              - sts:AssumeRole
+      Policies:
+        - PolicyName: "adf-retrieve-pipeline-definitions"
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "s3:GetObject"
+                  - "s3:GetObjectVersion"
+                  - "s3:List*"
+                Resource:
+                  - !Sub "${ADFDefinitionBucket.Arn}/*"
+                  - !Sub "${ADFDefinitionBucket.Arn}"
+        - PolicyName: "adf-retrieve-shared-modules"
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "s3:GetObject"
+                  - "s3:GetObjectVersion"
+                  - "s3:List*"
+                Resource:
+                  - !Sub "arn:${AWS::Partition}:s3:::${SharedModulesBucket}/*"
+                  - !Sub "arn:${AWS::Partition}:s3:::${SharedModulesBucket}"
+        - PolicyName: "adf-deploy-cloudformation-create"
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - cloudformation:CreateStack
+                Resource:
+                  - "*"
+                Condition:
+                  StringEquals:
+                    'aws:RequestTag/createdBy': "ADF"
+        - PolicyName: "adf-deploy-cloudformation-template"
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - cloudformation:CreateStack
+                  - cloudformation:DescribeStacks
+                  - cloudformation:UpdateStack
+                  - cloudformation:CreateChangeSet
+                  - cloudformation:DeleteChangeSet
+                  - cloudformation:DescribeChangeSet
+                  - cloudformation:ExecuteChangeSet
+                  - cloudformation:SetStackPolicy
+                  - cloudformation:ValidateTemplate
+                Resource:
+                  - "*"
+              - Effect: Allow
+                Sid: "PassRole"
+                Action:
+                  - 'iam:PassRole'
+                Resource: '*'
```

Comment:
I think that can be removed. (It's a carry over.) Can you just double check. I think the CodeBuild Project role will never need to  pass an IAM role

Reply:
I think it uses the PassRole when executing the CloudFormation stacks.
According to the change you introduced, it will always use one role, see https://github.com/awslabs/aws-deployment-framework/pull/424/files/7f9d1c95cba423d9e20533436652b6d172399ed3#diff-de3040c3b70530a31a46f3a9ce5ffde48ecf03da5153a64929720b8e53c334e6R58

The role is created and configured in this stack too, see https://github.com/awslabs/aws-deployment-framework/pull/424/files/7f9d1c95cba423d9e20533436652b6d172399ed3#diff-dead502985c4f4a35238ba4a93fcedfabd8d013bd70e545003066d4a01afe956R500-R501

So I think we can restrict it to:

```suggestion
                Resource:
                  - !GetAtt ADFPipelineMangementCloudFormationRole.Arn
```